pkg/kapp/diff/resource_with_history.go

```diff
@@ -153,10 +153,10 @@ func (r ResourceWithHistory) recalculateLastAppliedChange() ([]Change, string, s
 	// Continue to calculate historyless change with excluded fields
 	// (previous kapp versions did so, and we do not want to fallback
 	// to diffing against list resources).
-	recalculatedChange1, err := r.calculateChangePrev1(lastAppliedRes)
-	if err != nil {
-		return nil, "", "" // TODO deal with error?
-	}
+	// recalculatedChange1, err := r.calculateChangePrev1(lastAppliedRes)
```

Comment:
I will remove it..

Reply:
Could you also remove the comments for the removed method `// Continue to calculate historyless change...`